### DISCUSSION TITLE
The tooltip of the top button is not visible in setting of github project.

### DISCUSSION THREAD

**Discussion by inkyu0103** [community/community#50559] [Accessibility]
### Select Topic Area

Bug

### Body

I'm not sure if this is an accessibility issue. If not, I apologize in advance.

![image](https://user-images.githubusercontent.com/48270642/226528556-eac41730-e10d-4e86-b7eb-d8c1512fb701.png)

As shown above, there is a problem that the tooltip is not visible. 
I don't think it's a very critical issue, but I'm reporting it because I think there are users who will feel uncomfortable.

**Comment by cehfisher** [comment]
Hi @inkyu0103 - this appears to be resolved now. Could you please confirm and close out this issue? Thank you! (cc @queenofcorgis)
